docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -2,6 +2,27 @@
 
 A Service Locator is embedded in Lagom's development environment, allowing services to discover and communicate with each others. There are a number of settings and tasks available to tune the embedded Service Locator to your liking, let's explore them:
 
+## Default address
+
+By default, the service locator binds to `localhost`, but it is possible to use a different host by adding the following to your build.
+
+In the Maven root project pom:
+
+```xml
+<plugin>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-maven-plugin</artifactId>
+    <version>${lagom.version}</version>
+    <configuration>
+        <serviceLocatorAddress>0.0.0.0</serviceLocatorAddress>
+    </configuration>
+</plugin>
+```
+
+In sbt:
+
+@[service-locator-address](code/build-service-port.sbt)
```

Comment:
This references `code/build-service-port.sbt` but the actual file name is `code/build-services-ports.sbt` (note different pluralization)